### ISSUE TITLE
Error building on windows.

### ISSUE THREAD

**Issue by i1Fury** [tazz4843/whisper-rs#87]
I followed the BUILDING.md perfectly(i tested everything in the paths and all), yet ran into this error:
```
error: failed to run custom build command for `whisper-rs-sys v0.6.1 (C:\Users\Elliot\Desktop\clip_it_chat\whisper-rs\sys)`

Caused by:
  process didn't exit successfully: `C:\Users\Elliot\Desktop\clip_it_chat\whisper-rs\target\debug\build\whisper-rs-sys-a11b4329329d64f0\build-script-build` (exit code: 101)
  --- stdout
  cargo:rustc-link-search=C:\Users\Elliot\Desktop\clip_it_chat\whisper-rs\target\debug\build\whisper-rs-sys-2b53c4a775c6a71b\out
  cargo:rustc-link-lib=static=whisper
  cargo:rerun-if-changed=wrapper.h

  --- stderr
  thread 'main' panicked at 'Unable to find libclang: "the `libclang` shared library at C:/msys64/mingw64\\bin\\libclang.dll could not be opened: LoadLibraryExW failed"', C:\Users\Elliot\.cargo\registry\src\index.crates.io-6f17d22bba15001f\bindgen-0.64.0\./lib.rs:2393:31
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace```

**Comment by tazz4843** [comment]
I don't have Windows so can't really help. Quick internet search shows this: https://users.rust-lang.org/t/unable-to-find-libclang-the-libclang-issue/70746

Not sure if it helps, but might be worth a shot

**Comment by i1Fury** [comment]
It found libclang.dll just fine, its just panicking for some reason? libclang is in my msys2 bin folder which is in my path, same as gcc.

**Comment by i1Fury** [comment]
Also, if it helps, the Python version of Whisper installed through PiP works perfectly(although its much slower).

**Comment by tazz4843** [comment]
The Python version of whisper is completely unrelated. It shares zero code with whisper.cpp. See if you can build the upstream project: https://github.com/ggerganov/whisper.cpp

**Comment by tazz4843** [comment]
Gonna go ahead and close this, PR that might've fixed was merged a while ago and 0.11 is released with it.
